src/api/layers/ExternalGroupOfLayers.class.js

```diff
@@ -65,6 +68,7 @@ export default class ExternalGroupOfLayers extends ExternalLayer {
             isLoading = true,
             availableProjections = [],
             getFeatureInfoCapability = null,
+            maxResolution = DEFAULT_MAX_GEOADMIN_RESOLUTION,
```

Comment:
Can we not get the max resolution from the GetCapabilities of the external layers ? Otherwise this might not work for external layers.